source/includes/minimum-source-privileges.rst

```diff
@@ -0,0 +1,151 @@
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :widths: 10 30
+
+   * - Sync Type
+     - Minimum Source Privileges
+
+   * - Default
+     - .. code-block:: javascript
+
+          [
+	        {
+		        "resource": { "cluster": true },
+		        "actions": [
+			          "appendOplogNote"    ,
+			          "getDefaultRWConcern",
+			          "getShardMap"        ,
+			          "hostInfo"           ,
+			          "listDatabases"      ,
+			          "listShards"         ,
+			          "replSetGetConfig"   ,
+			          "replSetGetStatus"   ,
+			          "serverStatus"
+		          ]
+	      },
+	      {
+		        "resource": { "db": "", "collection": "" },
+		        "actions": [
+			          "changeStream"   ,
+			          "collStats"      ,
+			          "find"           ,
+			          "indexStats"     ,
+			          "listCollections",
+			          "listIndexes"
+		        ]
+	      },
+	      {
```

Comment:
[Additional spacing issues]

The items starting on line 38 need to have the same level of indentation throughout.